go/internal/ocxcli/doctor_coordinator.go

```
@@ -0,0 +1,278 @@
+	db, err := sql.Open("sqlite", "file:"+path+"?mode=rw")
+	if err != nil {
+		return "", doctorCoordinatorRecoveryError(err)
+	}
+	locked := false
+	defer func() {
+		if locked {
+			_, _ = db.Exec("ROLLBACK")
+		}
+		_ = db.Close()
+	}()
+	if _, err = db.Exec("PRAGMA busy_timeout = 0; BEGIN IMMEDIATE"); err != nil {
```

golangci-lint (2.13.2)

[error] (*database/sql.DB).Exec must not be called. use (*database/sql.DB).ExecContext

(noctx)